scripts/build-wheel-macOS.sh

```diff
@@ -35,10 +35,9 @@ cd ./build
 NUMPROC=$(sysctl -n hw.ncpu)
 echo "Running $NUMPROC parallel jobs"
 
-for PYBIN in /Users/runner/hostedtoolcache/Python/3.*/x64/bin/python3.[5-9]
+for PYBIN in /Users/runner/hostedtoolcache/Python/3.*/x64/bin/python3.*
 do
     cmake .. -DPYTHON_EXECUTABLE:FILEPATH=$PYBIN \
-             -DCMAKE_MACOSX_RPATH=ON \
```

Comment:
Default value of MACOSX_RPATH is TRUE (policy [CMP0042](https://cmake.org/cmake/help/latest/policy/CMP0042.html#policy:CMP0042))